ci/pipeline.yaml

```diff
@@ -151,6 +101,17 @@ resources:
       tag: latest
       username: iamapikey
       password: ((concourse-icr-containers-public.password))
+  - name: agent-operator-image-manifest-sha
+    type: key-value
+    icon: zip-disk
+  # - name: agent-operator-bundle-image
```

Comment:
should we remove this?

Reply:
I was actually wondering that no olm bundle image was created, but apparently, we did not need it in that pipeline so far, just the bundle yaml files. So yeah, let me remove that.